src/main/java/com/google/sps/data/Recommended.java

```diff
@@ -0,0 +1,86 @@
+package com.google.sps.data;
+
+import java.util.*;
+
+/*This class holds the TYDRecommended algorithm which will give the
+    user a recommended store to get their desired beverage from.
+*/
+public class Recommended {
+
+  public static Drink getDrink(String storeId, String drinkName) {
+    System.out.println(
+        "2"); // ---------------------------------------------------> REMOVE AFTER FINISHING TEST
+    List<Drink> drinkList = DrinkDAO.getDrinksByStore(storeId);
```

Comment:
What's the purpose of this function? It doesn't seem too intuitive to me that getDrinksByStore is called in Recommended.